app/src/main/java/com/example/thecodelab/service/GithubApi.java

```diff
@@ -7,6 +7,7 @@
 import retrofit2.http.GET;
 import retrofit2.http.Path;
 
+
 public interface GithubApi {
 
     @GET("search/users?q=type:User+location:Nairobi+language:JAVA")
```

Comment:
@NabunyaLilian there is a better way of doing this using retrofit's @QueryMap annotation. [(Link)](https://square.github.io/retrofit/2.x/retrofit/index.html?retrofit2/http/QueryMap.html)